doc.html

```diff
@@ -36,71 +44,96 @@ <h2>2 Terminology</h2>
 
 When used alone, the term "asset_tracking" refers to "asset_tracking ATLAS Service Template". Instead, "asset_tracking service" is equivalent to "an ATLAS Service implementing the asset_tracking ATLAS Service Template".
 
+<h3>2.1 GSIN</h3>
+The Global Shipment Identification Number (GSIN) is the GS1 Identification Key used to identify a logical grouping of logistic units for the purpose of a transport shipment from the seller to the buyer. The unique identifier is comprised of a GS1 Company Prefix, a Shipper Reference, and a Check Digit.
+
+<h3>2.2 SSCC</h3>
+The Serial Shipping Container Code (SSCC) is the GS1 identifier used to identify a logistic unit. This unique identifier is comprised of an Extension Digit, a GS1 Company Prefix, a Serial Reference, and a Check Digit.
+
+<h3>2.3 MAC Address</h3>
+Unique physical identifier of returnable asssets, including beacons and trackers, which are attachd to pallets and containers.
+
+GSIN and SSCC both are unique identifiers in GS1 standard which are generated and assigned to logistic units while a shipment is being created. Whereas, asset's identifier (MAC address) is a unique physical identifier which is hardcoded into the asset and never changes. Therefore, a container or pallet in mupltiple shipments during its lifetime can have multiple GSIN and SSCC, whereas, the returnable asset installed in the pallet or container will have only one MAC address.
+
 <h2>3 Pre-requisites</h2>
 
 A thorough understanding in the following is required for
 both service consumers or service provider implementors:
 <pre>
-    - GeoJSON specifications (<a href="https://geojson.org/">https://geojson.org/</a>)
+    - GeoJSON specifications (<a href="https://geojson.org/">https://geojson.org/</a>) 
```

Comment:
GeoJSON does not seem to be used in your service template. Why make it a pre-requisite?